integration_tests/src/main/python/orc_write_test.py

```diff
@@ -91,6 +91,20 @@ def test_write_round_trip(spark_tmp_path, orc_gens, orc_impl):
             data_path,
             conf={'spark.sql.orc.impl': orc_impl, 'spark.rapids.sql.format.orc.write.enabled': True})
 
+@pytest.mark.parametrize('orc_gens', orc_write_gens_list, ids=idfn)
+@pytest.mark.parametrize('orc_impl', ["native", "hive"])
+@allow_non_gpu(*non_utc_allow)
+def test_write_more_than_one_stripe_round_trip(spark_tmp_path, orc_gens, orc_impl):
```

Comment:
If the difference is just to generate more data than `test_write_round_trip`, why not just make this case generate more? 

And another question is if we want to generate more than one stripe for other cases in this file or just this case.

Reply:
Well, if the new case will fail I think it makes sense to add it and keep the previous one so we can test the two behaviours at the same time.

Another option might be to add the length to the `@pytest.mark.parametrize` too, but not sure if this will bring some complicated if else in `orc_gens` to xfail the failed cases.